app/controllers/admin/collections_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Admin
+  # Generates collection reports
+  class CollectionsController < ApplicationController
+    before_action :authenticate_user!
+    verify_authorized
+
+    def index
+      authorize! :admin_dashboard
```

Comment:
This should have its own policy

Reply:
UserController uses the admin dashboard policy: https://github.com/sul-dlss/happy-heron/blob/main/app/controllers/admin/users_controller.rb#L12

Since authorization will always and only be for admins, is it worth proliferating policies?

Reply:
I think it's getting very confusing becase we are adding aliases like `report?` that don't make sense in the case of a DashboardPolicy.